package.json

```diff
@@ -5,6 +5,8 @@
   "dependencies": {
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "@rematch/core": "^2.0.0-next.10",
+    "@rematch/loading": "^2.0.0-next.11",
```

Comment:
@mlh-tiennd cc: @mlh-binhnt @themitvp 
Hi Tien, at the admin panel template we'll use Redux-Toolkit. So in our admin panel projects too.

Reply:
I finished converting from rematch to redux-toolkit. Please review this. Thanks!